web/src/global.less

```diff
@@ -66,7 +66,6 @@ ol {
   }
 }
 
-// 兼容IE11
```

Comment:
use English comment will be better

Reply:
Hi, this comment just got removed from this PR.